azure_functions_worker/dispatcher.py

```diff
@@ -562,6 +581,7 @@ async def _handle__function_environment_reload_request(self, request):
 
             func_env_reload_request = \
                 request.function_environment_reload_request
+            directory = func_env_reload_request.function_app_directory
 
             # Append function project root to module finding sys.path
             if func_env_reload_request.function_app_directory:
```

Comment:
nit: reuse var above

Reply:
It is using the var above. directory is derived from func_env_reload_request